test-framework/provisioning/roles/barnacle/tasks/main.yml

```diff
@@ -35,22 +35,49 @@
       dest: /etc/resolv.conf
       mode: 0644
 
-  - name: "Report frontend-install.py URL"
-    debug:
-      msg: "https://raw.githubusercontent.com/Teradata/stacki/{{ lookup('env','GIT_BRANCH')|default('develop', True) }}/tools/fab/frontend-install.py"
+  - name: Create directory to mount stacki ISO
+    file:
+      path: /mnt/{{ lookup('env','STACKI_ISO')|basename }}
+      state: directory
 
-  - name: Download frontend-install.py
-    get_url:
-      url: "https://raw.githubusercontent.com/Teradata/stacki/{{ lookup('env','GIT_BRANCH')|default('develop', True) }}/tools/fab/frontend-install.py"
-      dest: /root/
-      mode: 0744
-    register: get_url_result
-    until: get_url_result is succeeded
-    retries: 6
-    delay: 10
+  - name: Mount the stacki ISO to access the required RPMs
+    command: mount /export/isos/{{ lookup('env','STACKI_ISO')|basename }} /mnt/{{ lookup('env','STACKI_ISO')|basename }}
```

Comment:
Feels like it would be cleaner to just mount the CD at /mnt/stacki so you wouldn't need to keep looking up the STACKI_ISO env variable.